core/src/main/java/org/elasticsearch/search/suggest/term/TermSuggester.java

```diff
@@ -97,5 +103,36 @@ private Token(Term term, int startOffset, int endOffset) {
         }
 
     }
+	
+	private void addExactMatch(TermSuggestionContext suggestion, IndexReader indexReader, Token token, TermSuggestion.Entry resultEntry) throws IOException{
```

Comment:
See the comment about using `TermsEnum#seekExact` instead of manually iterating through the term dictionary.